templates/Dockerfile-template.ubuntu

```diff
@@ -61,5 +62,10 @@ ENV         LD_LIBRARY_PATH=/usr/local/lib
 
 COPY --from=build /usr/local /usr/local/
 
+RUN \
+	apt-get update -y && \
+	apt-get install -y libva-drm1 libva1 && \
```

Comment:
runtime dependency ? 

Just for ubuntu ? That would explain the failure on https://travis-ci.org/jrottenberg/ffmpeg/builds/389354692 though.

Reply:
This should be move into the base container to avoid an extra layer